Readme.md

```diff
@@ -32,6 +32,26 @@ it "sets $foo on commit" do
 end
 ```
 
+### Temporary disable after commit hooks
+
+In your test_helper, you can specify the default
+
+```
+TestAfterCommit.enabled = true
+```
+
+Then use blocks in your tests to change the behavior:
+
+```
+TestAfterCommit.with_commits(true) do
```

Comment:
My preference is the way it is now since it describes what is actually happening. But I'll switch to whichever you prefer.